exporter/awsxrayexporter/internal/translator/aws.go

```diff
@@ -237,12 +238,12 @@ func makeAws(attributes map[string]pcommon.Value, resource pcommon.Resource, log
 		cwl = getLogGroupMetadata(configSlice, false)
 	}
 
-	if sdkName != "" && sdkLanguage != "" {
+	sdkLanguageSuffix := fmt.Sprintf(" for %s", sdkLanguage)
```

Comment:
nit, you don't really need `fmt.Sprintf()`.
```
sdkLanguageSuffix := " for " + sdkLanguage
```
would work, right?

Reply:
It does. I tend to prefer `fmt.Sprintf`, but you're right. Probably unnecessary for this.